common/tools/esm4mocha.mjs

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { readFile, stat, constants } from "node:fs/promises";
+import { dirname, join } from "node:path";
+import { fileURLToPath } from "url";
+import { Project } from "./dev-tool/node_modules/ts-morph/dist/ts-morph.js";
+
+function consoleLog(...args) {
+  if (process.env.DEBUG && process.env.DEBUG.startsWith("esm4mocha")) {
```

Comment:
This is fine, but I was curious if using the `debug` module would help automate these checks (and make them more robust - again probably not ever necessary but would be nice to do DEBUG=* or DEBUG=esm4mocha,express etc)

Reply:
For simplicity this file is standalone (without a package), so it doesn't have any 3rd-party dependencies. That is also why I imported from dev-tool's `ts-morph` directly.  That relative import path is causing dependency-testing to fail at the moment though.